docs/en/04_Changelogs/4.11.0.md

```diff
@@ -88,33 +83,35 @@ To view which dependencies require `silverstripe/graphql`, run this composer com
 
 `composer why silverstripe/graphql`
 
-### Tracking or ignoring the `.graphql-generated` and `public/_graphql` folders
+### Tracking or ignoring the `.graphql-generated` and `public/_graphql` folders {#graphqlv4-track-new-folders}
 
-Existing projects will not have an entry in their `.gitignore` file for `.graphql-generated` or `public/_graphql`. It is best practice for most situations to not track these folders in version control. You’ll have to manually add this entry to your `.gitignore`.
+Existing projects will not have an entry in their `.gitignore` file for `.graphql-generated` or `public/_graphql`. It is best practice for most situations to not track these folders in version control. You’ll have to manually add these entries to your `.gitignore` file.
 
-The `.gitignore` file in `silverstripe/installer` 4.11 has been updated to ignore both of these folders. If you start a new project from `silverstripe/installer` 4.11.0 and want to track the new folders, you’ll have to update your `.gitignore` file.
+The `.gitignore` file in `silverstripe/installer` 4.11.0 has been updated to ignore both of these folders. If you start a new project from `silverstripe/installer` 4.11.0 and want to track the new folders, you’ll have to remove those entries from your `.gitignore` file.
 
 ## Features and enhancements {#features-and-enhancements}
 
-### Upload and use WebP images {#webp}
+### Adding support for PHP 8.1 {#php81}
 
-WebP is an alternative image format for displaying picture on websites. It provides generally better results in most use cases to JPEG and PNG.
+The Silverstripe CMS recipe now officially supports PHP 8.1. This version of PHP introduced various deprecation warnings, most notably when passing null arguments to many native PHP functions. Silverstripe CMS now makes liberal use of the null coalescing operator when calling native PHP functions to convert null arguments to scalars. Other changes including adding the `__serialize` and `__unserialize` methods to classes implementing the [`Serializable`](https://www.php.net/manual/en/class.serializable) interface and adding the [`[#\ReturnTypeWillChange]`](https://www.php.net/manual/en/migration81.incompatible.php#migration81.incompatible.core.type-compatibility-internal) attribute to methods on classes implementing various PHP interfaces such as [`Iterator`](https://www.php.net/manual/en/class.iterator).
 
-Read [An image format for the Web](https://developers.google.com/speed/webp) for more information about WebP.
+### Dropping support for PHP 7.3 {#phpeol}
 
-WebP has wide – but not universal – support across web browsers. Internet Explorer is the main browser that does not support WebP at this stage.
+In accordance with our [PHP support policy](/getting_started/server_requirements#php), Silverstripe CMS Recipe 4.11.0 drops support for PHP 7.3. We expect to drop support for PHP 7 altogether around January 2023.
```

Comment:
I wonder if instead of "We expect to drop support for PHP 7 altogether around January 2023", we should call out the [major release policy RFC](https://github.com/silverstripe/silverstripe-framework/issues/10345)? It seems likely to me that CMS 4 will never drop PHP 7.4 support, and then CMS 5 will start with PHP 8.1 support.

Reply:
Seems weird putting an active RFC in a changelog. May just remove the big about expecting to drop PHP 7?